From 99acb975e2890ddb1328dc7e05840a2cb3c91d51 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Thu, 5 Nov 2020 20:46:35 +0100
Subject: [PATCH] busybox: enable whois by default

Some users of OpenWrt would like to use whois on their routers without
re-compiling whole busybox. It means that they need to know how to
cross-compile things for OpenWrt, enable it, compile busybox and flash
it on the router. That's can be difficult.

Tool whois can identify who owns a domain and how to get reach owner.
Providing this tool in OpenWrt someone does not need to use websites for
everything. According to config/networking/Config.in, it should take 6.3
kb.

Signed-off-by: Josef Schlehofer <pepe.schlehofer@gmail.com>
---
 package/utils/busybox/Config-defaults.in | 2 +-
 package/utils/busybox/Makefile           | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/package/utils/busybox/Config-defaults.in b/package/utils/busybox/Config-defaults.in
index 36731f2b6e..80bbe2708e 100644
--- a/package/utils/busybox/Config-defaults.in
+++ b/package/utils/busybox/Config-defaults.in
@@ -2567,7 +2567,7 @@ config BUSYBOX_DEFAULT_FEATURE_WGET_OPENSSL
 	default n
 config BUSYBOX_DEFAULT_WHOIS
 	bool
-	default n
+	default y
 config BUSYBOX_DEFAULT_ZCIP
 	bool
 	default n
diff --git a/package/utils/busybox/Makefile b/package/utils/busybox/Makefile
index 23dd5498a1..a12401ff30 100644
--- a/package/utils/busybox/Makefile
+++ b/package/utils/busybox/Makefile
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=busybox
 PKG_VERSION:=1.30.1
-PKG_RELEASE:=6
+PKG_RELEASE:=7
 PKG_FLAGS:=essential
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
-- 
2.25.1

